### PR TITLE
fix: minor fixes in Nodes and Tenants tables

### PR DESCRIPTION
**Pull request by artemmufazalov**

### CODE REVIEW

File: src/containers/Tenants/Tenants.js [hunk 1]
```diff
@@ -218,8 +219,8 @@ class Tenants extends React.Component {
                 },
                 accessor: ({Metrics = {}, CoresUsed}) => {
                     if (!isNaN(Number(CoresUsed))) {
-                        const cores = Math.round(Number(CoresUsed) * 100) / 100;
-                        return cores || '—';
+                        const cores = Number(CoresUsed).toFixed(2);
+                        return Number(cores) ? numeral(cores).format('0.00') : '—';
```
Comment (ariser) [inline src/containers/Tenants/Tenants.js:223]: what is the purpose of this change?
maybe extend [formatNumber](https://github.com/ydb-platform/ydb-embedded-ui/blob/main/src/utils/index.js#L58) with a precision argument and use it here?
there is also [formatCPU](https://github.com/ydb-platform/ydb-embedded-ui/blob/main/src/utils/index.js#L66), but looks like the orders of values don't match

Comment (artemmufazalov) [reply src/containers/Tenants/Tenants.js:223]: The purpose of this change to keep 2 values after a dot and display dash for zero values. I'm not sure that it makes sense to move that logic (display dash if zero) somewhere else

Comment (artemmufazalov) [reply src/containers/Tenants/Tenants.js:223]: The difference is that now it always displays 2 values after a dot, previously it was just rounded

Comment (ariser) [reply src/containers/Tenants/Tenants.js:223]: dash should remain here, of course
i'm talking about the formatting, there is a function that does that. It just lacks the precision argument, to specify these 2 fraction values. And it handles non-numeric values better than `if (Number(value)) {}`, you can just `formatNumber(value) || '—'`

Comment (ariser) [reply src/containers/Tenants/Tenants.js:223]: ah, zero should be a dash, i see

Comment (ariser) [reply src/containers/Tenants/Tenants.js:223]: my concern is currently `numeral` is only used in utils, and the rest of the app uses functions from utils to format values, this makes dependency change easy
in this PR you bring `numeral` to another module

Comment (ariser) [reply src/containers/Tenants/Tenants.js:223]: so all the logic that is going on here is fine
my comment is about formatting specifically

**Review by ariser — APPROVED** [review]
